### PR TITLE
Bug 1551904 - Deduplicate NetworkPolicy from availableKinds

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1551904

/assign @benjaminapetersen

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ phantomjsdriver.log
 .DS_Store
 test/test-results.xml
 npm-debug.log
+Chrome_*
```
Comment (benjaminapetersen) [inline .gitignore]: Just curious what creates `Chrome_*` files

Comment (spadgett) [reply .gitignore]: Our unit tests create those, at least for me.

Comment (benjaminapetersen) [reply .gitignore]: Top level? Not under `/test` or `/test/tmp`?  Weird.

Comment (spadgett) [reply .gitignore]: Yeah, top level.

```
~/git/origin-web-common master
❯ ls -l
total 616
drwxr-xr-x    3 sam  staff      96 Mar  6 08:09 Chrome_64.0.3282_(Mac_OS_X_10.13.3)
...
```

**Pull request merged**